### DISCUSSION TITLE
Forbidden resource link invitation users - Version 2.66.1

### DISCUSSION THREAD

**Discussion by dhikduk-io** [ToolJet/ToolJet#10598] [General]
Hi, please help,
There is a Forbidden resource error when entering the new invitation password
![Screenshot 2024-08-07 163640](https://github.com/user-attachments/assets/350dc32b-ca09-4

Thank u

**Comment by PriteshKiri** [comment]
Hey @dhikduk-io,

I’m having a little trouble seeing the image you shared—it's not visible on my end. Would you mind resending it when you get a chance?
